ui/apps/platform/src/Components/CVEStackedPill/SeverityStackedPill.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement } from 'react';
+
+import { vulnSeverityIconColors } from 'constants/visuals/colors';
+import { getPercentage } from 'utils/mathUtils';
+
+export type SeverityStackedPillType = {
```

Comment:
nit: we usually suffix the Props type for component with `...Props`

```suggestion
export type SeverityStackedPillProps = {
```

Reply:
Good eyes!